server/data/quotes.json

```diff
@@ -103,6 +103,10 @@
     "author": "Benjamin Whorf",
     "sentence": "Language is not simply a reporting device for experience but a defining framework for it."
   },
+  {
+    "author": "Bob Dylan",
+    "sentence": "A mistake is to commit a misunderstanding."
```

Comment:
Please add a quote regarding "learning a foreign language" somewhat similar.

Reply:
I have changed the quote